src/react-app-env.d.ts

```diff
@@ -0,0 +1,2 @@
+/// <reference types="react-scripts" />
+declare module 'styled-components';
```

Comment:
es una práctica común dejar un salto de línea al final del documento https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline